bin/extract_metrics/install.sh

```diff
@@ -0,0 +1,4 @@
+export AWS_ACCOUNT_ID=725604423866
```

Comment:
this AWS_ACCOUNT_ID is hard-coded because we only need it to install pulse?  am I reading that correctly?

Reply:
We could allow this to be an environment variable, I just hard-coded it for convenience. You can install p3d from either the prod or modl cluster but not the test cluster, so I decided to be explicit